### PR TITLE
Roll pub packages manually

### LINKED ISSUE #139861: Version solving failed for `intl: 0.19.0` with `flutter_localizations`
Hi,
Currently, the **flutter_localizations** package depends on **intl ^0.18.1**
Would it be possible to update this dependency to **intl ^0.19.0**

Have a nice day

### PR DESCRIPTION
**Pull request by loic-sharma**
The automated roll failed as a test needs to be updated: https://github.com/flutter/flutter/pull/145167

Fixes: https://github.com/flutter/flutter/issues/139861

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Data Driven Fixes]: https://github.com/flutter/flutter/wiki/Data-driven-Fixes

### CODE REVIEW

File: packages/flutter_localizations/test/material/translations_test.dart [hunk 1]
```diff
@@ -174,8 +174,8 @@ void main() {
     expect(localizations.selectedRowCountTitle(0), 'បាន​ជ្រើស​រើស​ធាតុ 0');
     expect(localizations.selectedRowCountTitle(1), 'បាន​ជ្រើស​រើស​ធាតុ 1');
     expect(localizations.selectedRowCountTitle(2), 'បាន​ជ្រើស​រើស​ធាតុ 2');
-    expect(localizations.selectedRowCountTitle(10000), 'បាន​ជ្រើស​រើស​ធាតុ 10.000');
-    expect(localizations.selectedRowCountTitle(123456789), 'បាន​ជ្រើស​រើស​ធាតុ 123.456.789');
+    expect(localizations.selectedRowCountTitle(10000), 'បាន​ជ្រើស​រើស​ធាតុ 10,000');
+    expect(localizations.selectedRowCountTitle(123456789), 'បាន​ជ្រើស​រើស​ធាតុ 123,456,789');
```
Comment (loic-sharma) [inline packages/flutter_localizations/test/material/translations_test.dart:178]: @goderbauer and I did some research:

1. Wikipedia article indicates Khmer uses commas as the thousands separator: https://en.wikipedia.org/wiki/Khmer_numerals#Numbers_from_100_to_10,000,000
2. Google Translate uses commas as the thousands separator: https://translate.google.com/?sl=en&tl=km&text=I%20have%201%2C000%20apples&op=translate
3. Gemini agrees it should be a comma: `The Khmer language uses a comma (,) as its thousands separator. However, you might also see spaces used as a thousands separator, especially in less formal contexts.`

**Review by goderbauer — APPROVED** [review]
LGTM

Thank you!

**Review by christopherfujino — APPROVED** [review]
ship it

File: packages/flutter_localizations/test/material/translations_test.dart (hunk 1 above)
Comment (ditman) [reply packages/flutter_localizations/test/material/translations_test.dart:178]: (Drive-by reviewing, feel free to ignore) ICU says: 

https://www.localeplanet.com/icu/km-KH/index.html

> Numbers
> * Grouping separator: `.`
> * Decimal separator: `,`

Sauce:

https://github.com/unicode-org/icu/blob/b396885aae1f515df35688ea72943583c0268836/icu4c/source/data/locales/km.txt#L17-L22

Comment (christopherfujino) [reply packages/flutter_localizations/test/material/translations_test.dart:178]: Sometimes software is too hard

Comment (christopherfujino) [reply packages/flutter_localizations/test/material/translations_test.dart:178]: Looks like this change is correct, per: https://github.com/flutter/flutter/pull/145167#issuecomment-1998170061

Comment (ditman) [reply packages/flutter_localizations/test/material/translations_test.dart:178]: Hah, and people complain about timezones :P What an amazing "Winning" column in that link :P

**Review by eliasyishak — DISMISSED** [review]
I need to make an update to analytics for `5.8.7`, can we wait until that publishes in a few minutes?

**Comment by loic-sharma** [comment]
@eliasyishak No problem. Let me know when you're good to go!

**Comment by christopherfujino** [comment]
For context, Elias and I believe that if we merge this as is, the version of unified_analytics here will make https://github.com/dart-lang/tools/issues/252 even worse, so we're going to try to roll this forward two hotfixes.

File: packages/flutter_localizations/test/material/translations_test.dart (hunk 1 above)
Comment (ditman) [reply packages/flutter_localizations/test/material/translations_test.dart:178]: This is where the separators flipped:

https://unicode.org/cldr/charts/44/delta/km.html

(Looks like ICU4c hasn't regenerated data from the latest CLDR yet?)

**Pull request merged**